webapp/src/test/java/io/github/microcks/util/DispatchCriteriaHelperTest.java

```diff
@@ -218,6 +220,24 @@ public void testBuildFromParamsMap() {
       assertEquals("?limitation=20?page=1", dispatchCriteria);
    }
 
+  @Test
+  public void testBuildFromParamsArrayMap() {
+    Multimap<String, String> paramsMap = ArrayListMultimap.create();
+    paramsMap.put("page", "1");
+    paramsMap.put("limit", "20");
+    paramsMap.put("limitation", "20");
+    paramsMap.put("status", "available");
+    paramsMap.put("status", "busy");
+
+    // 2 parameters should be taken into account, one with two values according to rules.
+    String dispatchCriteria = DispatchCriteriaHelper.buildFromParamsMap("page && status", paramsMap);
+    assertEquals("?page=1?status=available?status=busy", dispatchCriteria);
+
+    // 1 parameter with two values should be taken into account according to rules
+    dispatchCriteria = DispatchCriteriaHelper.buildFromParamsMap("status", paramsMap);
+    assertEquals("?status=available?status=busy", dispatchCriteria);
```

Comment:
Are the params values sorted in natural order? We need a way to be predictable regarding how a dispatch criteria is built either from the imported spec or from the incoming Http request.

Reply:
They should be in natural order, because i kept the use of a TreeMap to sort the keys, replacing the TreeMap by a MultiTreemap

Reply:
Oh right, the keys are sorted by natural order, but not the values, i'll check what i can do

Reply:
After checking the Multimap doc, it seems that both the values and the keys are ordered naturally by default 
![image](https://github.com/microcks/microcks/assets/48886723/610fa9ee-ab1b-4293-8fd0-289fb3f508ad)

Reply:
Excellent! Thank you.